### PR TITLE
{Release} Upgrade to Azure CLI 2.65.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,90 @@
...
+* Include `jq` in Azure Linux docker image (#29867)
+* Drop Debian 10 support (#29378)
+
+**Profile**
+
+* `az login`: Fall back to device code flow in GitHub Codespaces (#27443)
+
+**RDBMS**
+
+* `az postgres flexible-server update`: Support Case-Insensitive Input for `--tier`, `--performance-tier`, `--sku`, and `--maintenance-window` (#29840)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:82]:
```suggestion
* `az postgres flexible-server update`: Support case-insensitive input for `--tier`, `--performance-tier`, `--sku`, and `--maintenance-window` (#29840)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,90 @@
...
+
+* `az cosmosdb restore`: Add `--disable-ttl` to support for Restore with Time-To-Live Disabled (#29875)
+
+**dla**
+
+* `az dla`: Deprecate azure datalake analytics from CLI (#29879)
+
+**HDInsight**
+
+* `az hdinsight update`: Add parameter `--assign_identity_type` to allow to update to `SystemAssigned` identity. (#29816)
```
Comment (bebound) [inline src/azure-cli/HISTORY.rst:50]:
```suggestion
* `az hdinsight update`: Add parameter `--assign-identity-type` to allow to update to `SystemAssigned` identity. (#29816)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,90 @@
...
+* Fix #29849: `az containerapp job stop`: Fix stop execution response (#29907)
+* Fix #29166: `az containerapp compose create`: Support image with tag and support get ACR from image (#29933)
+
+**Cosmos DB**
+
+* `az cosmosdb restore`: Add `--disable-ttl` to support for Restore with Time-To-Live Disabled (#29875)
+
+**dla**
+
+* `az dla`: Deprecate azure datalake analytics from CLI (#29879)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:46]:
```suggestion
**DataLake**

* `az dla`: Deprecate datalake analytics (#29879)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,90 @@
...
+* Drop Debian 10 support (#29378)
+
+**Profile**
+
+* `az login`: Fall back to device code flow in GitHub Codespaces (#27443)
+
+**RDBMS**
+
+* `az postgres flexible-server update`: Support Case-Insensitive Input for `--tier`, `--performance-tier`, `--sku`, and `--maintenance-window` (#29840)
+* `az postgres flexible-server migration create` Add AWS_AURORA as a migration source type for PostgreSql (#29863)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:83]:
```suggestion
* `az postgres flexible-server migration create`: Add AWS_AURORA as a migration source type for PostgreSql (#29863)
```

**Review by evelyn-ys — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Pull request merged**